opentelemetry-sdk/src/resource/builder.rs

```diff
@@ -0,0 +1,102 @@
+use std::time::Duration;
+
+use opentelemetry::KeyValue;
+
+use super::{Resource, ResourceDetector};
+
+/// Builder to allow easy composition of a Resource
+#[derive(Debug)]
+pub struct ResourceBuilder {
+    resource: Resource,
+}
+
+impl Default for ResourceBuilder {
+    /// Create ResourceBuilder with [Resource::default()].
```

Comment:
can we mention what the default would be here also? i.e it'll contain the telemetry,env etc.?

Reply:
yeah I can add this here, I was thinking the link to the default builder might be good to not have possible duplication / stale comments

Reply:
added more info